.github/workflows/build-assets.yml

```diff
@@ -160,7 +160,7 @@ jobs:
         run: |
           mkdir -p ${{ needs.preamble.outputs.folder_build }}
           echo "${{ secrets.SIGNING_SECRET }}" > ${{ needs.preamble.outputs.folder_build }}/private.key
-          plugins=("CASSANDRAEMBED" "COUCHBASEEMBED" "ECLBLAS" "H3" "JAVAEMBED" "KAFKA" "MEMCACHED" "MYSQLEMBED" "NLP" "REDIS" "SQLITE3EMBED" "SQS" "PLATFORM")
+          plugins=("COUCHBASEEMBED" "H3" "JAVAEMBED" "KAFKA" "MEMCACHED" "MYSQLEMBED" "NLP" "REDIS" "SQLITE3EMBED" "SQS" "PLATFORM")
```

Comment:
Why are ECLBAS and CASSANDRAEMBED removed?

Reply:
Cassandraembed is removed because it's built as part of the platform. If you grab the 9.0.20-1 platform package from the website and extract it, you should see all the cassandra libraries present.

eclblas was removed in error. The initial problem was eclblas package not building the eclblas library, and the final package was just a collection of the vcpkg libraries.  Testing a change now and I'll push it shortly to fix this issue.

Reply:
I talked with Jim and the eclblas libraries are supposed to be in the platform (just like cassandraembed.)  Making the changes.